src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/ReportedPostsAndRepliesPage.java

```diff
@@ -2,11 +2,12 @@
 
 import com.wikia.webdriver.elements.mercury.components.discussions.common.ErrorMessages;
 import com.wikia.webdriver.elements.mercury.components.discussions.common.Post;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.SignInToFollowModalDialog;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)